devices/kintexu/familyInfo.xml

```diff
@@ -0,0 +1,5874 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<device_description>
+	<family>kintexu</family>
+	<site_types>
+		<site_type>
+			<name>SLICEL</name>
+			<is_slice/>
```

Comment:
Why don't we use these tags in the family builder to populate the site types in the classifications area?